### PR TITLE
Apothecary - FreeType 2.5.0 Formulae  

### PR DESCRIPTION
**Pull request by danoli3**
Formula's for:
- [x] OSX FAT (32 bit w/ libstdc++ and 64 bit w/ libc++)  [Build Status](https://travis-ci.org/danoli3/apothecary-den/branches)
- [x] iOS FAT (armv7, ~~armv7s~~~, arm64, i386, x86_64 all with libc++) [Build Status](https://travis-ci.org/danoli3/apothecary-den/branches)
- [x] Android - armeabi (@arturoc ?)
- [x] Android - armeabi-v7a (@arturoc ?)
- [x] Android - x86 (@arturoc ?)
- [ ] VS
- [ ] win_cb
- [x] Linux (32 bit)
- [x] Linux (64 bit)
- [x] Linux Armv6l
- [x] Linux Armv7l
- [x] emscripten

Notes:
-------------------------------
ofTrueTypeFont path fixes for FreeType 2.5.0+ are in a separate PR Here:

Original PR:
https://github.com/openframeworks/openFrameworks/pull/3152

### CODE REVIEW

**Comment by danoli3** [comment]
@bakercp Mind if we merge this soon? I'm looking to do the iOS update apothecary overhaul branch very soon... we only have 10 days till 64day.

**Comment by ofTheo** [comment]
@danoli3 I'm fine if you want to merge all the PRs for apothecary that are still missing platforms if that helps for you to move forward. Especially if its just changes to the apothecary formulas and not the libs themselves. 

We can do separate PRs for the remaining platforms.

**Comment by danoli3** [comment]
@ofTheo Okay yeah that will help a lot actually. I'll name the PR's accordingly then merge

File: scripts/apothecary/formulas/android_configure.sh [hunk 1]
```diff
@@ -16,11 +16,12 @@ export TOOLCHAIN_PATH=${NDK_ROOT}/toolchains/${TOOLCHAIN}/prebuilt/${HOST_PLATFO
...
 export CXX=${NDK_ROOT}/toolchains/${TOOLCHAIN}/prebuilt/${HOST_PLATFORM}/bin/${ANDROID_PREFIX}-g++
 export AR=${NDK_ROOT}/toolchains/${TOOLCHAIN}/prebuilt/${HOST_PLATFORM}/bin/${ANDROID_PREFIX}-ar
-export CFLAGS="--sysroot=${SYSROOT} -fno-short-enums"
 
+#export CFLAGS="--sysroot=${SYSROOT} -fno-short-enums"
 #export CFLAGS="$CFLAGS -I${SYSROOT}/usr/include/ -I${NDK_ROOT}/sources/cxx-stl/gnu-libstdc++/${TOOLCHAIN_VERSION}/include -I${NDK_ROOT}/sources/cxx-stl/gnu-libstdc++/${TOOLCHAIN_VERSION}/libs/${ABI}/include -I${NDK_ROOT}/sources/android/cpufeatures"
+export CFLAGS="-nostdlib --sysroot=${SYSROOT} -fno-short-enums"
 export CFLAGS="$CFLAGS -I${SYSROOT}/usr/include/ -I${NDK_ROOT}/sources/cxx-stl/gnu-libstdc++/include -I${NDK_ROOT}/sources/cxx-stl/gnu-libstdc++/libs/${ABI}/include"
-export LDFLAGS="--sysroot=${SYSROOT} -L${NDK_ROOT}/sources/cxx-stl/gnu-libstdc++/${TOOLCHAIN_VERSION}/libs/${ABI} -lz -lsupc++ -llog -ldl -lm -lc -lgnustl_static -lgcc"
+export LDFLAGS="--sysroot=${SYSROOT} -nostdlib -L${NDK_ROOT}/sources/cxx-stl/gnu-libstdc++/libs/${ABI}"
```
Comment (danoli3) [inline scripts/apothecary/formulas/android_configure.sh]: @arturoc Just keep in mind this change here may affect the other scripts for your Android ones, you may want to export the LDFLAGS later for any to overload this if it is different.

**Comment by danoli3** [comment]
Okay I think this is good to merge though. Adding it now.

**Pull request merged**